docs/_config.yml

```
@@ -42,15 +42,15 @@ remote_theme: rundocs/jekyll-rtd-theme@v2.0.10
 #   - vendor/ruby/
 
 # Document versioning
-version: master
+version: v0.6.0
```

this definitely needs to be master
```suggestion
version: master
```